### PR TITLE
feat: startup without mon

### LINKED ISSUE #1990: Startup OpenMLDB without `mon`
- startup OpenMLDB without `mon`
- check healthy

### PR DESCRIPTION
**Pull request by dl239**
- Start `openmldb` without `mon`
- Check  process after startup

### CODE REVIEW

**Review by vagetablechicken — APPROVED** [review]

File: release/bin/start.sh [hunk 1]
```diff
@@ -71,11 +72,30 @@ case $OP in
 
         # Ref https://github.com/tj/mon
         if [ "$COMPONENT" != "taskmanager" ]; then
-            $MON_BINARY "./bin/boot.sh $COMPONENT" -d -s 10 -l "$LOG_DIR"/"$COMPONENT"_mon.log -m "$OPENMLDB_PID_FILE";
-            sleep 1
+            ./bin/openmldb --flagfile=./conf/"$COMPONENT".flags --enable_status_service=true 2>&1 &
+            PID=$!
+            sleep 2
+            if kill -0 $PID > /dev/null 2>&1; then
+                /bin/echo $PID > "$OPENMLDB_PID_FILE"
```
Comment (aceforeverd) [inline release/bin/start.sh]: multiple nameservers conflicts the same pid file ?

File: release/bin/start.sh [hunk 2]
```diff
@@ -71,11 +72,30 @@ case $OP in
```
Comment (dl239) [inline release/bin/start.sh:73]: echo components

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Comment by lumianph** [comment]
[improve the startup script](https://trello.com/c/gpkR9ImA/58-improve-the-startup-script)

**Pull request merged**